src/sidebar/components/annotation-header.js

```diff
@@ -25,7 +25,7 @@ export default function AnnotationHeader({
   threadIsCollapsed,
 }) {
   const isCollapsedReply = isReply(annotation) && threadIsCollapsed;
-  const setCollapsed = useStore(store => store.setCollapsed);
+  const setExpanded = useStore(store => store.setExpanded);
```

Comment:
Here and in several other components/consumers, `setExpanded` makes more semantic sense than `setCollapsed` because almost all users are trying to expand, not trying to collapse. It also helps to delineate a little from the `thread.collapsed` property, which is informed by, but not entirely dictated by, the values in the resulting map.